ci_framework/roles/build_containers/molecule/default/converge.yml

```diff
@@ -19,5 +19,6 @@
   hosts: all
   vars:
     cifmw_build_containers_cleanup: true
+    cifmw_build_containers_config_file: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/roles/build_containers/molecule/default/containers.yaml"
```

Comment:
Once the file is moved, you can use `files/containers.yaml` instead for the path.

Reply:
(that file has to be seen by TCIB, can't use relative path - sorry, my bad)